app/views/admin/reservations/reservation_loans/_reservation_hold.html.erb

```diff
@@ -10,10 +10,10 @@
   <% reservation_hold.reservation_loans.each do |reservation_loan| %>
     <tr id="<%= dom_id(reservation_loan) %>">
       <% if reservation_loan.quantity %>
+        <td>*</td>
         <td><%= reservation_hold.item_pool.name %></td>
-        <td><%= reservation_loan.quantity %></td>
       <% else %>
-        <td><%= reservation_loan.reservable_item.id %></td>
+        <td><%= reservation_loan.reservable_item.number %></td>
```

Comment:
This is only somewhat related, but it makes it so the item names always appear in the same column. We don't have numbers for the "uncounted" items that are only represented by an ItemPool yet.